packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -139,30 +137,24 @@ class SwipeableDrawer extends React.Component {
         ? window.innerHeight - event.touches[0].clientY
         : event.touches[0].clientY;
 
-    if (!open) {
-      if (disableSwipeToOpen) {
-        return;
-      }
-      if (isHorizontal(this.props)) {
-        if (currentX > swipeAreaWidth) {
-          return;
-        }
-      } else if (currentY > swipeAreaWidth) {
-        return;
-      }
+    if (!open && disableSwipeToOpen) {
+      return;
     }
 
     nodeThatClaimedTheSwipe = this;
     this.startX = currentX;
     this.startY = currentY;
 
     this.setState({ maybeSwiping: true });
-    if (!open && this.paper) {
-      // the ref may be null when a parent component updates while swiping
-      this.setPosition(this.getMaxTranslate() + (disableDiscovery ? 20 : -swipeAreaWidth), {
-        changeTransition: false,
-      });
-    }
+    setImmediate(() => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate

Reply:
Meh… I'll change it to `setTimeout` then.